src/desktop/analytics/main_layout.js

```diff
@@ -3,25 +3,27 @@
 // or any other actions that occur on each page.
 //
 
-import { data as sd } from 'sharify'
-import { reportLoadTimeToVolley } from 'lib/volley'
+import { data as sd } from "sharify"
+import { reportLoadTimeToVolley } from "lib/volley"
 
 // Track pageview
-analytics.page(
-  { path: location.pathname },
-  { integrations: { Marketo: false } }
-)
+const pageType = window.sd.PAGE_TYPE || window.location.pathname.split("/")[1]
+var properties = { path: location.pathname }
+
+if (pageType == "artwork") {
+  properties["ecommerce"] = sd.ARTWORK.is_acquireable
+}
+
+analytics.page(properties, { integrations: { Marketo: false } })
 
 // Track pageload speed
 if (
   window.performance &&
   window.performance.timing &&
   sd.TRACK_PAGELOAD_PATHS
 ) {
-  window.addEventListener('load', function() {
-    const pageType =
-      window.sd.PAGE_TYPE || window.location.pathname.split('/')[1]
-    if (sd.TRACK_PAGELOAD_PATHS.split('|').includes(pageType)) {
+  window.addEventListener("load", function() {
+    if (sd.TRACK_PAGELOAD_PATHS.split("|").includes(pageType)) {
```

Comment:
My javascript scopes are rusty, but I'm assuming this can access the global `pageType` const declared above.